judge/forms.py

```diff
@@ -123,6 +123,18 @@ class ProposeProblemSolutionFormSet(inlineformset_factory(Problem, Solution, for
     pass
 
 
+class ProblemSuggestForm(ModelForm):
```

Comment:
why don't you use the `ProblemEditForm`?

Reply:
`ProblemSuggestForm` might be similar to `ProblemEditForm`, but cannot inherit even 10% of `ProblemEditForm`. Rewrite everything will be much simpler, as the EditForm needs improvements, while the SuggestForm seems to be enough.

Reply:
You can edit the `ProblemEditForm` (remove `authors` field, add `name` field)